Gemfile

```diff
@@ -61,7 +61,7 @@ gem 'uuid'
 gem 'will_paginate'
 
 # Prometheus exporter
-gem 'prometheus-client-mmap', '<= 0.28.1'
+gem 'prometheus-client-mmap', '~> 1.2'
```

Comment:
@mtclinton, was there any reason to upgrade `prometheus-client-mmap` dependency?

I haven't seen it as a requirement for `unleash`, so I'd suggest to take it out of scope of this PR.